packages/bundler/src/BundlerServer.ts

```diff
@@ -160,6 +166,24 @@ export class BundlerServer {
   async handleMethod (method: string, params: any[]): Promise<any> {
     let result: any
     switch (method) {
+      /** RIP-7560 specific RPC API */
+      case 'eth_sendTransaction':
+        if (!this.config.useRip7560Mode) {
+          throw new RpcError(`Method ${method} is not supported`, -32601)
+        }
+        if (params[0].sender != null) {
+          result = await this.methodHandlerRip7560.sendRIP7560Transaction(params[0])
+        // } else {
```

Comment:
no real need for "classes" `sendTransaction`, since it is a debug function only: it requires a local account, with is very long deprecated: sending legacy TXs is done with `eth_sendRawTransaction`

The question is only naming: should `eth_sendTransaction` is the method to send AA transactions.

Reply:
I am not 100% sure I understand what you prefer. 
1. `eth_sendRawTransaction` as the main method does not fit the RIP-7560 well as we don't really need to RLP-encode the transaction in the client. We can and nothing bad happens, but why bother?
2. `eth_sendTransaction` method implies **signing** the transaction request. I don't think it fits well.
3. `eth_sendRIP7560Transaction` method was added to be explicit that the signing has been done already. I don't know if the difference to the `eth_sendTransaction` method is worth the extra API method.

Reply:
eth_sendTransaction is for sending *unsigned* transaction, and is not a good candidate.
we need a new transaction signature.
eth_sendRip7560Transaction is good interim.
Maybe we should have a new namespace, e.g. "ethaa_" for the methods (sendTransaction, valdiate, estimate)

can also be a "sub-namespace", e.g "eth_aa_"

`getTransactionReceipt` and `getTransaction` should stay in normal "eth", namespace, Their input is hash, and the response is structured based on actual tx.